### PR TITLE
choeazycopy: Add version 1.0.0.23

### LINKED ISSUE #7936: [Request] choeazycopy
### Information

**Name:** choeazycopy

**Description:** ChoEazyCopy is Simple and powerful GUI to Microsoft RoboCopy tool.

**Homepage:** https://github.com/Cinchoo/ChoEazyCopy

**Download link(s):** https://github.com/Cinchoo/ChoEazyCopy/releases/download/v1.0.0.23/ChoEazyCopy.zip

**Some indication of popularity/repute:** ⭐ GitHub stars: 902

### PR DESCRIPTION
**Pull request by KaranKad**
Added ChoEazyCopy version 1.0.0.23
~Its a neutal application~
32bit application

Closes #7936

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by hu3rror** [comment]
It looks like this is a 64-bit program. I'm not entirely sure.

**Comment by KaranKad** [comment]
From https://github.com/Cinchoo/ChoEazyCopy/issues/14#issuecomment-757362783

> This application is build under 'Any CPU' mode, it means it runs in both 32 bit / 64 bit environments.

**Comment by rashil2000** [comment]
> It looks like this is a 64-bit program. I'm not entirely sure.

It is a 32bit GUI.

image

File: bucket/choeazycopy.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+    "version": "1.0.0.23",
+    "description": "ChoEazyCopy is Simple and powerful GUI to Microsoft RoboCopy tool.",
```
Comment (rashil2000) [inline bucket/choeazycopy.json:3]:
```suggestion
    "description": "Simple and powerful GUI to Microsoft's RoboCopy tool.",
```

File: bucket/choeazycopy.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    "homepage": "https://github.com/Cinchoo/ChoEazyCopy",
+    "license": "Unknown",
```
Comment (rashil2000) [inline bucket/choeazycopy.json:5]:
```suggestion
    "license": "CPOL-1.02",
```

File: bucket/choeazycopy.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    "url": "https://github.com/Cinchoo/ChoEazyCopy/releases/download/v1.0.0.23/ChoEazyCopy.zip",
+    "hash": "A01B3FB3BD098CE3FF4B549C5D18CB183A5F7D0BE5DD70D803ED6A465FEF7526",
+    "persist": "ChoEazyCopy.exe.config",
```
Comment (rashil2000) [inline bucket/choeazycopy.json:8]: I don't think this is user config file. Can you check the contents and confirm?

File: bucket/choeazycopy.json [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    "bin": "ChoEazyCopy.exe",
```
Comment (rashil2000) [inline bucket/choeazycopy.json:9]: Commandline args?

**Review by rashil2000 — CHANGES_REQUESTED** [review]

**Comment by rashil2000** [comment]
- We don't need to specify 32bit if 32bit is the only available build (you should revert the latest commit)
- Do you think we should create a new config file (if it doesn't exist) and persist it?

**Comment by KaranKad** [comment]
> * Do you think we should create a new config file (if it doesn't exist) and persist it?

No, I dont think there is any need to create a config file. The config file can exist anywhere in the system which contains input for robocopy command, the config file has a dedicated extension which can be executed to launch ChoEazyCopy with that specific settings.

**Pull request merged**